python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py

```diff
@@ -0,0 +1,285 @@
+            primary_index = self.m1m3_actuator_ids.index(actuator_id)
+            if secondary_exist:
+                secondary_index = self.m1m3_secondary_actuator_ids.index()
+                await self.checkpoint(
+                    f"Running bump test on DAA FA ID: {actuator_id} "
+                    f"(Primary/Secondary: {primary_index}/{secondary_index})"
+                )
+            else:
+                await self.checkpoint(
+                    f"Running bump test on SAA FA ID: {actuator_id} "
+                    f"(Primary {primary_index})"
+                )
+
+            try:
+                await self.mtcs.run_m1m3_actuator_bump_test(
+                    actuator_id=actuator_id,
+                    primary=True,
+                    secondary=secondary_exist,
+                )
+            # raise a RuntimeError if the bump test fails
+            # and continue loop to next actuator_id
+            except RuntimeError:
+                failed_actuators.append(actuator_id)
+                self.log.exception(f"Failed to run bump test on FA ID {actuator_id}.")
+
+            # Getting test status
+            (
+                fa_id,
+                primary_status,
+                secondary_status,
+            ) = await self.mtcs.get_m1m3_bump_test_status()
+
+            # BumpTest.PASSED = 6
+            # BumpTest.FAILED = 7
+            if not secondary_exist:
+                # Getting primary indexes for failures
+                if primary_status == BumpTest.FAILED:
+                    failed_primary_index.append(primary_index)
+
+                self.log.info(
+                    f"Bump test done for {i + 1} of {len(self.actuators_to_test)}"
+                    f"Bump test on FA ID {fa_id}. Primary FA: "
+                    f"{primary_status.upper()}."
+                )
+
+            else:
+                # Getting secondary indexes for failures
+                if secondary_status == BumpTest.FAILED:
+                    failed_secondary_index.append(secondary_index)
+
+                self.log.info(
+                    f"Bump test done for {i + 1} of {len(self.actuators_to_test)}"
+                    f"Bump test on FA ID {fa_id}. Primary/Secondary: "
+                    f"{primary_status.upper()}/{secondary_status.upper()} ."
+                )
+
+        end_time = time.monotonic()
+        elapsed_time = (end_time - start_time) / 60.0
+
+        # Final checkpoint
+        await self.checkpoint(
+            f"M1M3 bump test completed. It took {elapsed_time:.2f} minutes"
+        )
+
+        # Final message with bump test results/status
+        if not failed_actuators:
+            self.log.info("All actuators PASSED the bump test.")
+        else:
+            error_message = f"Actuators {failed_actuators} FAILED the bump test. \n "
+            error_message += f"Primary Indexes: {failed_primary_index} \n "
+            error_message += f"Secondary Indexes: {failed_secondary_index}"
```

Comment:
The message is a bit misleading, mixing IDs and indices. This might cause a lot of troubles looking for the actuator. See above for one option.